### PR TITLE
Add ssl root anchor script

### PR DESCRIPTION
**Pull request by pgporada**
## Pull Request Checklist

**Is this in reference to an existing issue?**
No it is not.

#### General

- [X] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [X] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [X] RuboCop passes
(on the check and the test)

- [x] Existing tests pass 

#### New Plugins

- [X] Tests

- [X] Add the plugin to the README

- [X] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

#### Purpose

I need to verify that a website is using a specific root certificate anchor. This script allows me to do that.

#### Known Compatablity Issues
You need awk installed. This is essentially a wrapper script. Any tips on how to remove that dependency and do it in ruby would be awesome.

### CODE REVIEW

File: bin/check-ssl-anchor.rb [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  def validate_opts
+    config[:servername] = config[:host] unless config[:servername]
+  end
+
+  # Do the actual work and massage some data
+  def anchor_information
+    data = `openssl s_client \
+                -connect #{config[:host]}:#{config[:port]} \
+                -servername #{config[:servername]} < /dev/null 2>&1 | \
+            awk '/Certificate chain/,/---/'`.split(/$/).map(&:strip)
```
Comment (eheydrick) [inline bin/check-ssl-anchor.rb]: instead of the awk you could do `.match(/Certificate chain(.*)---\nServer certificate/m)[1]`

**Comment by eheydrick** [comment]
Thanks for the contribution @pgporada. Could you add an entry to the CHANGELOG.md under Unreleased and we'll get this merged.

File: bin/check-ssl-anchor.rb (hunk 1 above)
Comment (pgporada) [reply bin/check-ssl-anchor.rb]: Thanks for the tip. I think I've managed to successfully implement your suggestion.

**Comment by pgporada** [comment]
All done 👍

**Comment by eheydrick** [comment]
Did you push? Don't see the updates.

**Comment by pgporada** [comment]
Well that was embarrassing. Sorry about that.

**Comment by eheydrick** [comment]
No worries, thanks again for the contribution! I'll get a new version out soon.

**Comment by eheydrick** [comment]
Released in 1.4.0.

**Pull request merged**